django/utils/inspect.py

```diff
@@ -16,13 +16,18 @@ def _get_callable_parameters(meth_or_func):
     return _get_func_parameters(func, remove_first=is_method)
 
 
+arg_kinds = frozenset(
+    {
+        inspect.Parameter.POSITIONAL_ONLY,
+        inspect.Parameter.KEYWORD_ONLY,
+        inspect.Parameter.POSITIONAL_OR_KEYWORD,
+    }
+)
+
+
 def get_func_args(func):
```

Comment:
There are no tests for get_func_args(), should we add some?

Reply:
`get_func_args()` is a small wrapper that is indirectly tested in `migrations.test_writer`. I'm not sure it's worth adding more tests for it.